recipe/session/index.js

```diff
@@ -13,6 +13,7 @@
  * under the License.
  */
 "use strict";
+"use client"; // Important for NextJS support (SessionAuth is a client component)
```

Comment:
we can't do this.

Reply:
We should create a separate file `/recipe/session/clientcomponents/sessionauth.js`